routing/routing_tests/index_graph_test.cpp

```diff
@@ -503,4 +525,51 @@ UNIT_CLASS_TEST(RestrictionTest, LoopGraph)
   double constexpr kExpectedRouteTimeSec = 3.48;
   TestRouteTime(*m_starter, AStarAlgorithm<IndexGraphStarter>::Result::OK, kExpectedRouteTimeSec);
 }
+
+UNIT_TEST(IndexGraph_OnlyTopology_1)
+{
+  using Edge = TestIndexGraphTopology::Edge;
+  // Add edges to the graph in the following format: (from, to, weight).
+
+  uint32_t const numVertices = 5;
+  TestIndexGraphTopology graph(numVertices);
+  graph.AddDirectedEdge(0, 1, 1.0);
+  graph.AddDirectedEdge(0, 2, 2.0);
+  graph.AddDirectedEdge(1, 3, 1.0);
+  graph.AddDirectedEdge(2, 3, 2.0);
+
+  double const expectedWeight = 2.0;
+  vector<Edge> const expectedEdges = {{0, 1}, {1, 3}};
+
+  TestTopologyGraph(graph, 0, 3, true /* pathFound */, expectedWeight, expectedEdges);
+  TestTopologyGraph(graph, 0, 4, false /* pathFound */, 0.0, {});
+}
+
+UNIT_TEST(IndexGraph_OnlyTopology_2)
+{
+  using Edge = TestIndexGraphTopology::Edge;
+
+  uint32_t const numVertices = 1;
+  TestIndexGraphTopology graph(numVertices);
+  graph.AddDirectedEdge(0, 0, 100.0);
+
+  double const expectedWeight = 0.0;
+  vector<Edge> const expectedEdges = {};
+
+  TestTopologyGraph(graph, 0, 0, true /* pathFound */, expectedWeight, expectedEdges);
+}
+
+UNIT_TEST(IndexGraph_OnlyTopology_3)
+{
+  using Edge = TestIndexGraphTopology::Edge;
```

Comment:
I didn't want to do it because there are other tests in this file.
On the other hand, they don't use the word "Edge", so done.